modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-annotating-workloads_{context}"]
+
+= Annotating workloads
+
+Two annotations must be available to isolate a workload into partitioned resources. Annotate the containing namespace with `workload.openshift.io/allowed` and the pod
```

Comment:
nit: "Two annotations must be specified to isolate partitioned workloads from each other."  That's a complete guess.  My interest is to point out that so far, the reader has encountered "workload partitions", "user workloads", and "cluster management workloads" and when I read "partitioned resources" I am unsure if you mean something different.

If "partitioned resources" are different, please introduce the term in an earlier concept module.